### PR TITLE
generic rules for some japanese websites

### LINKED ISSUE #124797: www.pref.saga.lg.jp
### Issue URL (Ads)
 [https://www.pref.saga.lg.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.pref.saga.lg.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.0.181
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Japanese

### LINKED ISSUE #124798: www.city.kumamoto.jp
### Issue URL (Ads)
 [https://www.city.kumamoto.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.city.kumamoto.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.0.181
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Japanese

### LINKED ISSUE #124800: www.city.unzen.nagasaki.jp
### Issue URL (Ads)
 [https://www.city.unzen.nagasaki.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.city.unzen.nagasaki.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.0.181
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by DandelionSprout**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [ ] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

* Fix #124800
* Fix #124797
* Fix #124798

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

Upon analysis last night, it worries me that most `www.city.*.jp` sites would use *completely* different element trees and names, such that finding recurring patterns on more than 3 reported sites proved futile. And even then, `##a[rel=nofollow][href*="/ban/pub/default.aspx?"]` was one of only three 3-site patterns I could find.

Additionally, I'm adding `###fBanArea.floating` for the sake of `https://www.pref.saga.lg.jp/`

2. Screenshots

N/A (Covered by the original threads)

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by zloyden — APPROVED** [review]
Please also add some screenshots for the pull request

**Comment by DandelionSprout** [comment]
>Please also add some screenshots for the pull request
Copied over from the original reports:

image

image

image

>zloyden requested a review from Alex-302

Funny you should say that, because Alex has been mass-closing issue reports for similar Japanese sites in the past couple days without actually fixing them.

File: JapaneseFilter/sections/general_elemhide.txt [hunk 1]
```diff
@@ -130,6 +130,8 @@
 ##div[class^="google-2ad"]
 ##div[id^="active_overlay"]
 ##div[id^="post-"][class*="post-"] > div[id*="-"][class*="%e"]
+##a[rel="nofollow"][href*="/ban/pub/default.aspx?"]
```
Comment (Alex-302) [inline JapaneseFilter/sections/general_elemhide.txt:133]: On some sites it will hide owl links.

Comment (zloyden) [reply JapaneseFilter/sections/general_elemhide.txt:133]: I guess we can add it and see how it goes.

Comment (Alex-302) [reply JapaneseFilter/sections/general_elemhide.txt:133]: Nah. No one reports that sites.

Comment (zloyden) [reply JapaneseFilter/sections/general_elemhide.txt:133]: Ok.

**Pull request closed without merging**